### ISSUE TITLE
Fetch episode specific data

### ISSUE THREAD

**Issue by kaustubhhiware** [kaustubhhiware/rose#3]
It should be possible to fetch information for only one episode. That is, add functionality for another parameter, call it `-e` or `--episode` which allows to fetch information for that particular episode only.

Usage: python scrape_views.py -s 'Two and a half men' -e 'S03E02'

If the contributor wants, they could also handle this case (not a priority) : 

python scrape_views.py -e -s 'Two and a half men S03E02'

The expected output is IMDB rating, US views if available and a link to the episode. 

Pointed out by Shalini Mukhopadhyay

**Comment by kunalnpandey** [comment]
I would like to work on fetching episode specific data

**Comment by kaustubhhiware** [comment]
@kunalnpandey you should not start working before the issue is formally assigned to you. I see that you've claimed the issue, but please be a bit more patient for the future contributions. It won't make a difference in a small project like this, but it's always helpful to be courteous.

Marking this issue as being assigned to you.

**Comment by kaustubhhiware** [comment]
Closed with #6.
